.husky/pre-commit-phpcbf.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+# Disable exit on error since we handle exit codes manually
+set +e
+
+# Get list of staged PHP files
+STAGED_PHP_FILES=$(git diff --cached --name-only --diff-filter=d | grep ".php$") || true
```

Comment:
If you're going to `set +e`, you can skip the `|| true` here.

Also you may want to sync this `git diff` with the earlier one.

Reply:
111a5b59897f64c21894db32791daf579a400cb4